saleor/graphql/order/tests/test_fulfillment.py

```diff
@@ -711,14 +711,18 @@ def test_fulfillment_update_tracking_send_notification_false(
 
 
 CANCEL_FULFILLMENT_MUTATION = """
-    mutation cancelFulfillment($id: ID!, $warehouseId: ID!) {
+    mutation cancelFulfillment($id: ID!, $warehouseId: ID) {
         orderFulfillmentCancel(id: $id, input: {warehouseId: $warehouseId}) {
```

Comment:
We should also add a test for running this mutation without the `input` parameter.